.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,29 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/request_review')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Add label and request review
+        run: |
+          reviewers="cryostatio/reviewers"
+
+          echo "Adding label 'review-requested' to the PR"
+          gh pr edit ${{ github.event.issue.number }} --add-label "review-requested"
+
+          echo "Requesting review from @cryostatio/reviewers team"
```

Comment:
`echo "Requesting review from @$reviewers"` perhaps, just so that the string isn't repeated?